### PR TITLE
neovim: add missing dependency

### PR DESCRIPTION
**Pull request by subnut**
v0.10.0 added requirement of markdown treesitter parser.

### CODE REVIEW

**Comment by subnut** [comment]
The tests for inline-markdown's extensions seems to be failing even though I have enabled all extensions via the patch...

@nekopsykose any idea what the problem might be?

**Comment by nekopsykose** [comment]
you cant update query because neovim depends on a specific version

File: contrib/tree-sitter-markdown/patches/extensions.patch
Comment (nekopsykose) [inline contrib/tree-sitter-markdown/patches/extensions.patch:1]: you can just these env vars instead of patching it at all

Comment (subnut) [reply contrib/tree-sitter-markdown/patches/extensions.patch:1]: I tried, it didn't work

(although this doesn't work either, lol)

File: contrib/neovim/template.py [hunk 1]
```diff
@@ -28,7 +28,8 @@
 depends = [
     "lua5.1-lpeg",
     "tree-sitter-lua",
-    "tree-sitter-query",
+    "tree-sitter-markdown",
+    "tree-sitter-query<0.4.0",
```
Comment (nekopsykose) [inline contrib/neovim/template.py:32]:
```suggestion
    "tree-sitter-query",
```

don't need explicit versions

File: contrib/tree-sitter-markdown/template.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+pkgver = "0.2.3"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_check_target = "test"
+hostmakedepends = [
+    "gmake",
+    "nodejs",
+    "tree-sitter-cli",
+    "pkgconf",
```
Comment (nekopsykose) [inline contrib/tree-sitter-markdown/template.py:11]:
```suggestion
    "pkgconf",
    "tree-sitter-cli",
```

File: contrib/neovim/template.py (hunk 1 above)
Comment (subnut) [reply contrib/neovim/template.py:32]: right. forgot to remove this after adding `ignore = True`.

removed :+1:

**Pull request merged**